api/src/catalog/catalog.utils.ts

```diff
@@ -92,39 +96,68 @@ export async function fetchCatalog(
   // (if overwrite = true or if file does not exist)
   const processSeasons = seasons.seasons.map(async (season) => {
     const seasonCode = season.season_code;
-    const outputPath = `${STATIC_FILE_DIR}/catalogs/${seasonCode}.json`;
+    const fullCatalogPath = `${STATIC_FILE_DIR}/catalogs/${seasonCode}.json`;
+    const publicCatalogPath = `${STATIC_FILE_DIR}/catalogs/public/${seasonCode}.json`;
+    let catalogFull: Catalog = { computed_listing_info: [] };
+    let catalogPublic: Catalog = { computed_listing_info: [] };
 
+    // Fetch and save the full catalog (including evaluations)
     if (
-      !overwrite &&
-      (await fs.access(outputPath).then(
+      overwrite ||
+      !(await fs.access(fullCatalogPath).then(
```

Comment:
maybe this should go back to 
`!overwrite &&
      (await fs.access(outputPath).then(`